src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml

```diff
@@ -539,9 +539,9 @@ components:
         suspSntcProbationDurtnTxt:
           type: string
         ticketedAmt:
-          type: double
+          type: string
```

Comment:
Are these text fields in Oracle?

Reply:
This change is when the string is getting converted to numeric.  The datatype is still numeric in the application, but we were getting errors with Jackson specifically if mapped here.  After Jackson, Mapstruct converts this ticketedAmt column to numeric.